.github/workflows/test-integration.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Checkout openconext-devconf
         uses: actions/checkout@v4
-        with: 
+        with:
```

Comment:
Personally I would not include this change in the PR. This change seems unrelated to the changeset, was it autoformatting? Or maybe it is good old boyscout work.

Feel free to leave it and resolve this feedback by yourself.

Reply:
It's auto formatting, I cannot reinstate the space, so gonna leave it out.